.github/PULL_REQUEST_TEMPLATE.MD

```diff
@@ -0,0 +1,10 @@
+Please review our [Contribution Guidelines](https://github.com/avinoth/elixir-companies/blob/master/CONTRIBUTING.md) to ensure your changes abides by it.
+
+Make sure your changes ticks off the below checklist.
+
+- [ ] Company Name
```

Comment:
Couple thoughts on this list:
- For consistency let's only capitalize the first word like you did in the last 2.  
- We should add something about their GitHub link
- Do we need the last one?  I'd hope that is a given.

What about something like:

- [ ] Company name
- [ ] Company description
- [ ] GitHub link
- [ ] How Elixir is used
- [ ] Sorted alphabetically